firebase-tools appdistribution:distribute command is failing

<!-- DO NOT DELETE
validate_template=true
template_path=.github/ISSUE_TEMPLATE/bug_report.md
-->


###  Environment info


Using Node

firebase-tools: 13.27.0

Platform: macOS


### Test case

N/A

### Steps to reproduce

  npx -y firebase-tools appdistribution:distribute ${Build Directory} --debug --app ${App location} --groups 'QAs' --release-notes 'Test'


### Expected behavior

Should distribute to testers/groups successfully

### Actual behavior

node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module 'js-yaml'
Require stack:
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/apphosting/yaml.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/apphosting/config.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/apphosting/config.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/apphosting/serve.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/apphosting/index.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/controller.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/commandUtils.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/commands/database-get.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/commands/index.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/index.js
- /Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/bin/firebase.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/apphosting/yaml.js:8:16)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
    at Module.require (node:internal/modules/cjs/loader:1311:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/apphosting/yaml.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/apphosting/config.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/apphosting/config.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/apphosting/serve.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/apphosting/index.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/controller.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/emulator/commandUtils.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/commands/database-get.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/commands/index.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/index.js',
    '/Users/administrator/.npm/_npx/7750544ccf494d8b/node_modules/firebase-tools/lib/bin/firebase.js'
  ]
}

Node.js v20.18.1
Error: Process completed with exit code 1.

--- a/package.json
+++ b/package.json
@@ -129,7 +129,7 @@
     "google-auth-library": "^9.11.0",
     "inquirer": "^8.2.6",
     "inquirer-autocomplete-prompt": "^2.0.1",
-    "js-yaml": "^4.1.0",
+    "js-yaml": "^3.14.1",
     "jsonwebtoken": "^9.0.0",
     "leven": "^3.1.0",
     "libsodium-wrappers": "^0.7.10",

--- a/package.json
+++ b/package.json
@@ -129,6 +129,7 @@
     "google-auth-library": "^9.11.0",
     "inquirer": "^8.2.6",
     "inquirer-autocomplete-prompt": "^2.0.1",
+    "js-yaml": "^4.1.0",
     "jsonwebtoken": "^9.0.0",
     "leven": "^3.1.0",
     "libsodium-wrappers": "^0.7.10",

